assets/src/edit-story/components/canvas/page.js

```diff
@@ -72,16 +85,67 @@ function Page() {
 			{ hasSelection && (
 				<Selection { ...selectionProps } />
 			) }
+			{ displayMoveable && (
+				<Moveable
+					target={ document.querySelector( '.selection' ) }
+					pinchThreshold={ 20 }
+					draggable={ true }
+					resizable={ true }
+					rotatable={ true }
+					onDrag={ ( { target, beforeTranslate } ) => {
+						frame.translate = beforeTranslate;
+						target.style.transform = `translate(${ beforeTranslate[ 0 ] }px, ${ beforeTranslate[ 1 ] }px)`;
+					} }
+					onDragStart={ ( { set } ) => {
+						set( frame.translate );
+					} }
+					onDragEnd={ () => {
+						setPropertiesOnSelectedElements( { x: selectionProps.x + frame.translate[ 0 ], y: selectionProps.y + frame.translate[ 1 ] } );
+					} }
+					onResizeStart={ ( { target, set, setOrigin, dragStart } ) => {
+						// @todo
+					} }
+					onResize={ ( { target, width, height, drag } ) => {
+						target.style.width = `${ width }px`;
```

Comment:
As far as I've seen it's using `width` and `height` in guidelines and has a separate scaling feature instead in addition to resizing. In case of some elements (such as image) which should always keep their ratio and only have resizing handles in the corners (based on Figma) then using scalable instead might be an option.

Will check the options.

Reply:
I looked around Moveable - I think width/height for now are good enough - they do a lot of mutations themselves that would bust layout. We can circle back to this later if we find a better way.